halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/providers/AbstractProviderCommand.java

```diff
@@ -31,18 +32,17 @@
 
   protected abstract String getProviderName();
 
-  public Account getAccount(String accountName) {
-    DaemonService service = Daemon.getService();
-    String currentDeployment = service.getCurrentDeployment();
-    ObjectMapper mapper = new ObjectMapper();
-    String providerName = getProviderName();
-    return mapper.convertValue(
-            service.getAccount(currentDeployment, providerName, accountName, !noValidate),
-            Providers.translateAccountType(providerName)
-    );
+  @Override
+  public String getCommandName() {
+    return getProviderName();
+  }
+
+  @Override
+  public String getDescription() {
+    return "Manage Spinnaker configuration for the " + getProviderName() + " provider";
   }
 
-  public Provider getProvider() {
+  private Provider getProvider() {
```

Comment:
It's only used by this command to determine which provider to ask for, and how to cast it. If I wind up needing this as `protected`, I'll increase the access level.